### PR TITLE
add START pool

### PR DESCRIPTION
**Pull request by best-coder-NA**

### CODE REVIEW

**Review by SarjuHansaliya — APPROVED** [review]
LGTM

File: src/constants.ts [hunk 1]
```diff
@@ -60,6 +60,7 @@ export const STAKING_ADDRESSES = [
   '0x10E5d5f598abb970F85456Ea59f0611D77E00168', // AVAX-ELE
   '0xfd0824dF1E598D34C3495e1C2a339E2FA23Af40D', // AVAX-FRAX
   '0x76Ad5c64Fe6B26b6aD9aaAA19eBa00e9eCa31FE1', // AVAX-FXS
+  '0x5105d9De003fB7d22979cd0cE167Ab919E60900A', // AVAX-START
```
Comment (HariSeldon23) [inline src/constants.ts:64]: @best-coder-NA Any reason for the whitespace?

**Review by HariSeldon23 — APPROVED** [review]
Small pedantic change to clean up whitespace. But we can add a linter to our process in the future @SarjuHansaliya - something for you to think about

**Comment by SarjuHansaliya** [comment]
@HariSeldon23 I think this whitespace is to separate AVAX pools and PNG pools

**Comment by HariSeldon23** [comment]
> @HariSeldon23 I think this whitespace is to separate AVAX pools and PNG pools

Fair enough. Any idea why the Test is failing? I tried to rerun it and it still fails

**Comment by SarjuHansaliya** [comment]
> Fair enough. Any idea why the Test is failing? I tried to rerun it and it still fails
yeah due to 0 APR

**Comment by bmino** [comment]
Yup the line breaks are intentional to separate pool "groups"

**Review by bmino — APPROVED** [review]

**Comment by bmino** [comment]
Addressing failed tests in #36

**Pull request merged**